app/views/dashboard/show.html.erb

```diff
@@ -7,10 +7,24 @@
 		</h4>
 	</div>
 <% end %><br><br><br><br>
+<div class="center-align">
+
+	<!-- <a class="waves-effect waves-light btn-small #c5e1a5 light-green lighten-3 black-text">
```

Comment:
This has been moved to the location I think we will have it but it is commented out for now until we can implement the functionality